### ISSUE TITLE
"Global" variables can't be assigned

### ISSUE THREAD

**Issue by pmorch** [FF-Remote-Control/FF-Remote-Control#2]
Posted on http://groups.google.com/group/mozilla.dev.extensions/browse_frm/thread/6cacc96b42a04e73 :

Remote Control can't set/get "global" variables properly. E.g.

foo={a:23, b:34}

Sets window.foo on the page, but not when run in the extension.
However, if foo already exists like above, (e.g. set through firebug),
then

foo.a = 83

run through Remote Control *does' change the global foo. Also,

window.foo={a:23, b:34}

does work. (!?)

So basically variable assigning is flaky I guess.

**Comment by nneonneo** [comment]
Seems to be fixed now; running `foo={a:23, b:34}` via remote control properly creates `window.foo` visible to the web developer tools.
